source/meta.tex

```diff
@@ -3906,7 +3906,7 @@
 
 \indexlibraryglobal{type_of}%
 \begin{itemdecl}
-consteval bool type_of(info r);
+consteval info type_of(info r);
```

Comment:
There's another pull request that also fixes this one.

Reply:
> There's another pull request that also fixes this one.

Do I need to take any action (i.e. revert this change)?